app/connector/BindingTariffClassificationConnector.scala

```diff
@@ -183,4 +186,22 @@ class BindingTariffClassificationConnector @Inject() (
       client.GET[Seq[ReportResult]](url)
     }
 
+  def updateUser(o: Operator)(implicit hc: HeaderCarrier): Future[Operator] =
+    withMetricsTimerAsync("update-user") { _ =>
+      val url = s"${appConfig.bindingTariffClassificationUrl}/users/${o.id}"
+      client.PUT[Operator, Operator](url = url, body = o)
+    }
+
+  def getUserDetails(id: String)(implicit hc: HeaderCarrier): Future[Operator] =
+    withMetricsTimerAsync("update-user") { _ =>
+      val url = s"${appConfig.bindingTariffClassificationUrl}/users/$id"
+      client.GET[Operator](url = url)
+    }
```

Comment:
Really quick thing - it's possible for this function to be called for an ID that does not exist by force-browsing to `/users/user/999` for example, so it would be better if this function returned `Future[Option[User]]` and the caller presented an error page when it is `None`.

I have updated it in this other PR because there I needed to know whether a user already existed or not when creating users so no need to change it here :) 
https://github.com/hmrc/tariff-classification-frontend/pull/495/files#diff-e61cb87110d42f6bd01268a9d96e0618f296ec385dd98d227a95c2abf11a8367R192
https://github.com/hmrc/tariff-classification-frontend/pull/495/files#diff-3e2d2bf4e14c1df3b447923bcbcd32b040b09cd37ad2dabed08666428278e70fR49

Reply:
great, thanks!